src/plugins/intel_cpu/src/nodes/executors/convolution_implementations.cpp

```diff
@@ -256,6 +274,18 @@ const std::vector<ExecutorImplementation<ConvAttrs>>& getImplementations() {
             AcceptsAnyShape<ConvAttrs>,
             CreateDnnlDefault<DnnlConvolutionPrimitive, ConvAttrs>{}
             )
+        OV_CPU_INSTANCE_ACL(
+            "convolution_acl_lowp", ExecutorType::Acl, OperationType::Convolution,
+            // supports
+            [](const ConvConfig& config, [[maybe_unused]] const MemoryFormatFilter& memoryFormatFilter) -> bool {
+                VERIFY(isQuantized(config), UNSUPPORTED_SRC_PRECISIONS);
+                VERIFY(config.attrs.postOps.size() <= 1U, UNSUPPORTED_BY_EXECUTOR);
```

Comment:
those verify check are supposed to add extra logic on top of executor limitations.
Are those executor limitations only?
If so, it is better to add 'supports' method to acl_conv.cpp

Reply:
Added `ACLConvolutionExecutor::supports` method